tests/pipelines/test_pipelines_common.py

```diff
@@ -83,16 +83,20 @@ def test_vae_slicing(self):
 
         inputs = self.get_dummy_inputs(device)
         inputs["prompt"] = [inputs["prompt"]] * image_count
+        if "image" in inputs:  # fix batch size mismatch in I2V_Gen pipeline
+            inputs["image"] = [inputs["image"]] * image_count
         output_1 = pipe(**inputs)
 
         # make sure sliced vae decode yields the same result
         pipe.enable_vae_slicing()
         inputs = self.get_dummy_inputs(device)
         inputs["prompt"] = [inputs["prompt"]] * image_count
+        if "image" in inputs:
+            inputs["image"] = [inputs["image"]] * image_count
         inputs["return_dict"] = False
         output_2 = pipe(**inputs)
 
-        assert np.abs(output_2[0].flatten() - output_1[0].flatten()).max() < 3e-3
```

Comment:
Here when I was testing I2V Gen Pipeline, I found that this pipeline result is not  that reproducible, i.e. even with same  inputs both from ` inputs = self.get_dummy_inputs(device)` and disable any changes like `pipe.enable_vae_slicing()`
by comment it out, the result still failed to pass the check. But I have not figure out why. @yiyixuxu Any idea?